config/manager/manager.yaml

```diff
@@ -30,6 +30,8 @@ spec:
           env:
             - name: RELATED_IMAGE_WASMSHIM
               value: "oci://quay.io/kuadrant/wasm-shim:latest"
+            - name: TOPOLOGY_NAMESPACE
+              value: "kuadrant-system"
```

Comment:
```suggestion
              valueFrom:
                fieldRef:
                  fieldPath: metadata.namespace
```

WDYT?

Reply:
This is the more robust approach, we do loss the ability to switch of the reconcile of the topology configmap easily with using the make targets. It would mean editing the deployment to change or remove that entry. Which I think is okay for now.